src/shared_utilities/FileReader.jl

```diff
@@ -456,7 +456,7 @@ function get_data_at_date(
         outfile_root,
         Dates.DateTime(0), # dummy date
         varname,
-        comms_ctx,
+        space,
     )
     return nans_to_zero.(field)
```

Comment:
I haven't tested it without the `nans_to_zero`. Maybe we don't need it anymore now that we're using the temporal albedo file with NaNs replaced by 1. I agree that we want to handle NaNs differently for different cases, rather than just setting them to 0. 

Is it okay if I open an issue to look into this in a separate PR? It's hard to test ClimaLand from the coupler right now because of the bug fixed here and also dependency updates I need to propagate

Reply:
Actually I was able to remove `nans_to_zero` altogether. If all of our tests pass I think it should be fine in the coupler too.

Reply:
ok! awesome.